cypress/e2e/marc-authority/manual-linking/linking-bibField240-to-authority110-with-subfield-t.cy.js

```diff
@@ -0,0 +1,149 @@
+      testData.userProperties = createdUserProperties;
+
+      marcFiles.forEach((marcFile) => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
+          () => {
+            DataImport.verifyUploadState();
+            DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+            JobProfiles.waitLoadingList();
+            JobProfiles.search(marcFile.jobProfileToRun);
+            JobProfiles.runImportFile();
+            JobProfiles.waitFileIsImported(marcFile.fileName);
+            Logs.checkStatusOfJobProfile('Completed');
+            Logs.openFileDetails(marcFile.fileName);
+            for (let i = 0; i < marcFile.numOfRecords; i++) {
+              Logs.getCreatedItemsID(i).then((link) => {
+                createdAuthorityIDs.push(link.split('/')[5]);
+              });
+            }
+          },
+        );
+      });
+      cy.login(testData.userProperties.username, testData.userProperties.password, {
+        path: TopMenu.inventoryPath,
+        waiter: InventoryInstances.waitContentLoading,
+      });
+    });
+  });
+
+  after('Deleting created user', () => {
+    cy.getAdminToken().then(() => {
+      Users.deleteViaApi(testData.userProperties.userId);
+      InventoryInstance.deleteInstanceViaApi(createdAuthorityIDs[0]);
+      createdAuthorityIDs.forEach((id, index) => {
+        if (index) MarcAuthority.deleteViaAPI(id);
+      });
+    });
+  });
+
+  it(
+    'C374111 Link the "240" of "MARC Bib" field with "110" field with a $t of "MARC Authority" record. (spitfire) (TaaS)',
+    { tags: ['extendedPath', 'spitfire'] },
+    () => {
+      InventoryInstance.searchByTitle(createdAuthorityIDs[0]);
+      InventoryInstances.selectInstance();
+      InventoryInstance.editMarcBibliographicRecord();
+      QuickMarcEditor.updateExistingField(testData.tag240, testData.newTag240Content);
+      QuickMarcEditor.checkContent(testData.newTag240Content, 11);
+      InventoryInstance.verifyAndClickLinkIcon(testData.tag240);
+      MarcAuthorities.switchToSearch();
+      InventoryInstance.verifySelectMarcAuthorityModal();
+      InventoryInstance.verifySearchOptions();
+      InventoryInstance.searchResults(testData.authority110FieldValue);
+      InventoryInstance.clickLinkButton();
+      QuickMarcEditor.verifyAfterLinkingAuthority(testData.tag240);
+      QuickMarcEditor.checkUnlinkTooltipText(testData.tag240, 'Unlink from MARC Authority record');
+      QuickMarcEditor.checkViewMarcAuthorityTooltipText(bib240AfterLinkingToAuth110[0]);
+      QuickMarcEditor.verifyTagFieldAfterLinking(...bib240AfterLinkingToAuth110);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkAfterSaveAndClose();
+      InventoryInstance.waitInventoryLoading();
+      InventoryInstance.verifyAlternativeTitle(
+        0,
+        1,
+        `${testData.authorityIconText}${marcFiles[0].instanceAlternativeTitle}`,
+      );
+      InventoryInstance.goToMarcApp();
```

Comment:
Or you can use this method "InventoryInstance.clickViewAuthorityIconDisplayedInInstanceDetailsPane()" to open the marc file.

Reply:
Thanks! I've deleted new function